🔧(labels) add missing 'release' label

jmaupetit opened the pull request:
## Purpose

When submitting a PR for a release, there is no real label that fits with this purpose.

## Proposal

- [x] add the `release` label with `FUN` label color

Fixes #5

jmaupetit committed:
🔧(labels) add missing 'release' label

When submitting a PR for a release, there is no real label that fits
with this purpose.

lunika commented on github/labels/default.json:
```diff
@@ -47,6 +47,10 @@
   },
+  {
+    "name": "release",
+    "color": "006b75"
```
It's the same color as `FUN` label, is it what you want ?

jmaupetit replied:
Yes! Unless you have another suggestion?

lunika replied:
Yes something green, maybe like `196811` ?

jmaupetit replied:
![](https://media.giphy.com/media/Hq2XDestpCXq8/giphy.gif)

lunika replied:
![](https://media.giphy.com/media/x7gjmBuaHrWak/giphy.gif)

jmaupetit replied:
Done.

jmaupetit committed:
🐳(labels) mount github configuration as a volume

We do not want to build a new image whenever we update labels; hence
using a volume when invoking the labels script seems more appropriate.
